tests/e2e/local/minikube/install_prereqs.sh

```diff
@@ -1,12 +1,27 @@
 #!/bin/bash
 
 case "${OSTYPE}" in
-  darwin*) sh install_prereqs_macos.sh;;
```

Comment:
Can we refactor these lines and reuse them as a function?

Reply:
Sure. I just pushed an update, PTAL. Thanks!